frappe/desk/listview.py

```diff
@@ -71,9 +71,23 @@ def get_group_by_count(doctype: str, current_filters: str, field: str) -> list[d
 		group_by=f"`tab{doctype}`.{field}",
 		fields=["count(*) as count", f"`{field}` as name"],
 		order_by="count desc",
-		limit=50,
```

Comment:
On second thought, things like FC might have lakhs of users in some documents :pensive: 

Maybe filtering first ~1000 is good enough?

Reply:
added this change